app/models/profile.rb

```diff
@@ -12,6 +12,7 @@ class Profile < ApplicationRecord
   belongs_to :account, optional: true
   belongs_to :business_objective, optional: true
   belongs_to :benchmark, class_name: 'Xccdf::Benchmark'
+  belongs_to :parent_profile, class_name: 'Profile', optional: true
```

Comment:
Just to confirm, it's only optional on "canonical profiles" right?

Reply:
Correct. I was thinking canonical profiles are `parent_profile == nil`